bowlingGame.js

```diff
@@ -0,0 +1,65 @@
+class BowlingGame {
+  constructor(){
+    this.rolls = [];
+    this.currentFrame = 1;
+  };
+  roll(number){
+    if(number > 10){
+      throw "Invalid Roll";
+    }
+    else {
+      this.rolls.push( { value: number, frame: this.currentFrame } );
+    };
+    
+  };
+  // [10, 5, 5, 10, 5, 3]
+  score(){
+    let totalScore = 0;
+    let currentRoll = 0;
+    while(currentRoll < this.rolls.length){
+      if(this.currentFrame == 10){
+        totalScore += this.sumTheRest(currentRoll);
+        break;
+      }
+      else if(this.frameIsStrike(currentRoll)){       
+        totalScore += this.bonusScoring(currentRoll);
```

Comment:
I think that this bonusScoring method is correct for the strike scenario. I like how it uses 'this.rolls[currentRoll + 1].value' to access the next rolls value. I'll probably end up trying to implement something like this in my code.